test/test-applications/aspnet/Samples.AspNetMvc5/Samples.AspNetMvc5.csproj

```diff
@@ -143,18 +143,14 @@
     <Folder Include="Models\" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\..\..\src\Datadog.Trace\Datadog.Trace.csproj">
```

Comment:
Do we still need this reference? You were able to remove it in `Samples.AspNetMvc4`.

Reply:
Yes we still need it here because we have a `DistributedTracingApiController` that tests distributed tracing and needs to access `Span` properties at compile-time. In the future we can make this reflection-based if we need to trim it out, but for now it's okay

Reply:
Ok, thanks!